ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml

```diff
@@ -72,6 +72,12 @@
       tags: [build_tools]
     - role: gcc_11                # JDK19+
       tags: [build_tools]
+    - role: devkit                # JDK19+
+      devkit_version: gcc-11.3.0-Centos7.9.2009-b01
```

Comment:
The devkit needs to be build "b02", as I have been doing lots of testing on b01, and wanted a clean/known build that is essential for reproducible builds. Also aarch64 needs to be Centos7.6 devkit as for reproducible builds by RHEL users, only aarch64 rhel 7.6 is available.
So the versions need to be:
- x64/ppc64le : gcc-11.3.0-Centos7.9.2009-b02
- aarch64 : gcc-11.3.0-Centos7.6.1810-b02

Reply:
Done